content/webapp/services/prismic/transformers/events.ts

```diff
@@ -217,7 +228,7 @@ export function transformEvent(
   ];
   const secondaryLabels = [...eventInterpretations];
 
-  return { ...event, labels, primaryLabels, secondaryLabels };
+  return { ...event, type: 'events', labels, primaryLabels, secondaryLabels };
```

Comment:
Moving the type down is to satisfy TypeScript – if the `type` is in the `event` variable enough, it's seen as type `string`, but TS expects it to be an enum value.